modular-docs-manual/content/topics/module_anchor-and-file-names-concept.adoc

```diff
@@ -41,15 +41,20 @@ The format defined here is recommended because it is the most stable and versati
 For more information about Asciidoc anchors, see the link:http://asciidoctor.org/docs/user-manual/#anchordef[Asciidoctor User Manual].
 --
 
-File names:: Give the module file the same name as the anchor used in it (which is the same as or similar to the module heading), also separated by dashes. Add a prefix with an underscore to the file name to indicate the module type in the format `prefix_file-name`. Use `con_` for concept, `ref_` for reference, `proc_` for procedure, and `assembly_` for assembly.
```

Comment:
@vikram-redhat, @tradej, @rkratky, since we're leaving that note in, which makes it clear that these file naming suggestions are strictly recommendations only, I think we should at least mention that adding module affixes is a consideration, and leave the example file names as they were. Then teams can just disregard if they like. Not requiring it is good, but not even mentioning it would be a pity for those who could get value from it but haven't thought of it. I know @tradej, @rkratky, and my teams have found great value in it, although others may not.

If we do go with this change as is, then these file name examples really aren't helpful, and in fact are troublesome if you look at the concept and assembly files, which are now identical - this will not be permitted by any authoring tool.

I'd propose the following for this section:

```
File names:: Give the module file the same name as the anchor used in it (which is the same as or similar to the module heading). Assembly and module file names should accurately and closely reflect the title of the assembly or module.
+
Consider adding affixes to file names to distinguish module types, if needed. This may improve the organization and navigability of your files depending on the structure and needs of your project. For example, you might add a prefix to the file name to indicate the module type in the format `prefix_file-name`, where `con_` is for concept, `ref_` is for reference, `proc_` is for procedure, and `assembly_` is for assembly.
+
.Examples
* `con_guided-decision-tables.adoc`  (Concept module)
* `proc_creating-guided-decision-tables.adoc`  (Procedure module for creating)
* `proc_editing-guided-decision-tables.adoc`  (Procedure module for editing)
* `ref_guided-decision-table-examples.adoc`  (Reference module with examples)
* `ref_guided-decision-table-columns.adoc`  (Reference module with column types)
* `assembly_guided-decision-tables.adoc`  (Assembly of guided decision table modules)

[NOTE]
====
These guidelines for file names are _recommended_. Project teams may adapt or deviate from these guidelines as needed.
====
```

Reply:
I agree with this comment.  Why not offer suggestions?